client/verta/tests/test_permissions/test_visibility_e2e.py

```diff
@@ -1,247 +0,0 @@
-"""
-End-to-end tests for org permissions access and actions.
-
-"""
-import pytest
-
-import requests
-
-from verta._internal_utils import _utils
-from verta.visibility import (
-    OrgCustom,
-    Private,
-)
-from verta.dataset import Path
-from verta.environment import Python
-
-
-pytestmark = pytest.mark.not_oss
-
```

Comment:
@evgeniilezhnin Instead of deleting this file completely, could we add

```python
pytest.skip("visibility deprecated; tests disabled for permissions v2", allow_module_level=True)
```

? I'd like to check later to see if we can adjust these tests for v2.

Reply:
added